apps/www/pages/storage/Storage.tsx

```diff
@@ -56,7 +56,10 @@ function StoragePage() {
             </span>,
           ]}
           subheader={[
-            'An open source Object store with unlimited scalability, for any file type.',
+            <>
+              An open source <b>S3 Compatible</b> Object Store, with unlimited scalability, for any
```

Comment:
```suggestion
              An open source <strong className="text-foreground font-normal">S3 Compatible</strong> Object Store, with unlimited scalability, for any
```

Reply:
Sorry, smol nit. We usually don't use bold but rather a more prominent color.
And using `<strong ...>` as the html tag makes it more semantically (and SEO) relevant.

Reply:
(btw accepting this suggestion might make prettier fail)